src/server/Document/index.jsx

```diff
@@ -19,7 +19,9 @@ const renderDocument = async ({
   service,
   url,
 }) => {
-  const cache = createCache();
+  const cache = createCache({
+    key: 'bbc',
```

Comment:
were we missing this all this time?

Reply:
In emotion 10 it seemed to just be defaulted (to `css`) - in v11 we now need to specify it